### PR TITLE
Update full page examples

### PR DESCRIPTION
**Pull request by hannalaakso**
Some updates to full page templates for the accessibility audit:
- Updated "What is your nationality page" to contain a form within details component for the accessibility audit.
- Added "What is the last country you visited?" example.

### CODE REVIEW

File: app/views/full-page-examples/what-was-the-last-country-you-visited/index.njk [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+              value: "france",
+              text: "France"
+          },
+          {
+              value: "spain",
+              text: "Spain"
+          },
+          {
+              value: "italy",
+              text: "Italy"
```
Comment (nickcolley) [inline app/views/full-page-examples/what-was-the-last-country-you-visited/index.njk]: Should we have an 'other' given that we don't have a complete list?

File: app/views/full-page-examples/what-is-your-nationality/index.njk [hunk 2]
```diff
@@ -62,8 +63,11 @@
             {
               id: "confirm-nationality",
               value: "british-nationality",
-              text: "British (including English, Scottish, Welsh or from Northern Ireland)",
-              checked: values["confirm-nationality"]|length and "british-nationality" in values["confirm-nationality"]
+              text: "British",
+              checked: values["confirm-nationality"]|length and "british-nationality" in values["confirm-nationality"],
```
Comment (nickcolley) [inline app/views/full-page-examples/what-is-your-nationality/index.njk]: I think the second checkbox we can move the text in brackets into hint text too.

Comment (hannalaakso) [reply app/views/full-page-examples/what-is-your-nationality/index.njk]: 👍

File: app/views/full-page-examples/what-was-the-last-country-you-visited/index.njk (hunk 1 above)
Comment (hannalaakso) [reply app/views/full-page-examples/what-was-the-last-country-you-visited/index.njk]: 👍

**Pull request merged**